src/status_im2/contexts/quo_preview/profile/profile_card/view.cljs

```diff
@@ -0,0 +1,110 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.view
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im2.contexts.quo-preview.profile.profile-card.style :as style]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]))
+
+(defn profile-card
+  [{:keys [show-sign-profile? key-card? profile-picture? name hash
+           emoji-hash on-press-dots? on-press-sign?]
+    :or   {show-sign-profile? false
+           key-card?          false}}]
+  [rn/view
+   {:style style/card-container}
+   [rn/view
+    {:style style/card-header}
+    [user-avatar/user-avatar
+     {:full-name         name
+      :profile-picture   profile-picture?
+      :size              :medium
+      :status-indicator? false
+      :ring?             true}]
+    [quo/button
+     {:size     32
+      :type     :blur-bg
+      :icon     true
+      :on-press on-press-dots?}
+     :i/more]]
+   [rn/view
+    {:style style/name-container}
+    [quo/text
+     {:size            :heading-2
+      :weight          :semi-bold
+      :number-of-lines 1
+      :style           style/user-name} name]
+    (when key-card?
+      (quo/icon
+       :i/keycard-full))]
+   [quo/text
+    {:weight          :monospace
+     :number-of-lines 1
+     :style           style/user-hash} hash]
+   (when emoji-hash
+     [quo/text
+      {:weight          :monospace
+       :number-of-lines 1} emoji-hash])
+   (when show-sign-profile?
+     [button/button
+      {:on-press             on-press-sign?
+       :type                 :community
+       :community-color      colors/turquoise-60
+       :community-text-color colors/white
+       :style                style/sign-button} "Sign in to this profile"])])
+
+
+(def descriptor
+  [{:label "Show: Sign in to this profile?"
+    :key   :show-sign-profile?
+    :type  :boolean}
+   {:label "Show: Is from key card?"
+    :key   :key-card?
+    :type  :boolean}
+   {:label "Hash"
+    :key   :hash
+    :type  :text}
+   {:label "Name"
+    :key   :name
+    :type  :text}
+   {:label "Emoji hash"
+    :key   :emoji-hash
+    :type  :text}])
+
+(def mocked-profile-picture
```

Comment:
Please put mock image in `resources/images/mock`. [Example](https://github.com/status-im/status-mobile/blob/915b8ebd9a38ce59c1039088475d4cde70b44e66/src/status_im2/contexts/quo_preview/switcher/switcher_cards.cljs#L95)

Reply:
I will create a clj file for the base64 image since I can't use images like .png for this case because I need to use `uri` in the `rn/image` so local files are not an option. I hope that's fine.

Reply:
I've messaged you over Discord about this